src/Command/GenerateSitemapCommand.php

```diff
@@ -98,12 +98,12 @@ private function executeChannel(ChannelInterface $channel, OutputInterface $outp
             $xml
         );
 
-        $output->writeln(\sprintf('Finished generating sitemap index for channel "%s" at path "%s"', $channel->getCode(), $path));
+        $output->writeln(\sprintf('Finished generating sitemap index for channel "%s" at path "%s"', $channel->getCode() ?? 'no_code', $path));
     }
 
     private function path(ChannelInterface $channel, string $path): string
     {
-        return \sprintf('%s/%s', $channel->getCode(), $path);
+        return \sprintf('%s/%s', $channel->getCode() ?? 'no_code', $path);
```

Comment:
Same here (and other places in file) - this could even conflict in case multiple channels dont have codes